eventstore/serializers.py

```diff
@@ -397,3 +424,28 @@ class Code(serializers.Serializer):
 
 class ForgetContactSerializer(serializers.Serializer):
     contact_id = serializers.UUIDField(required=True)
+
+
+class RegistrationSerializer(serializers.ModelSerializer):
```

Comment:
Should we not leave this one in the registration app? its not used in eventstore